src/test/java/com/radixdlt/atommodel/validators/ValidatorConstraintScryptTest.java

```diff
@@ -89,6 +89,22 @@ public void when_validating_registered_validator_particle_with_no_address__resul
 		assertThat(staticCheck.apply(validator).getErrorMessage()).contains("address");
 	}
 
+	@Test
+	public void when_validating_registered_validator_particle_with_invalid_url__result_has_error() {
+		RegisteredValidatorParticle validator = mock(RegisteredValidatorParticle.class);
+		when(validator.getAddress()).thenReturn(mock(RadixAddress.class));
+		when(validator.getUrl()).thenReturn("this_is_not_a_valid_url");
+		assertThat(staticCheck.apply(validator).getErrorMessage()).contains("url");
+	}
+
+	@Test
+	public void when_validating_registered_validator_particle_with_valid_url__result_has_no_error() {
+		RegisteredValidatorParticle validator = mock(RegisteredValidatorParticle.class);
+		when(validator.getAddress()).thenReturn(mock(RadixAddress.class));
+		when(validator.getUrl()).thenReturn("http://www.radixdlt.com/");
+		staticCheck.apply(validator);

```

Comment:
Should the result be checked here to ensure it has no error?

Reply:
It should! Good catch. I forgot that we have switched from exceptions to return values here some time ago.